### PR TITLE
feat: add venue and schedule_item api

### PR DESCRIPTION
**Pull request by 180909**

### CODE REVIEW

File: backend/bamboo/schemas/venue.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    name = String(required=True, validate=Length(0, 100))
+    address = String(required=True, validate=Length(0, 100))
+    city_id = Integer(required=True)
+
+
+class VenueOut(Schema):
+    id = Integer()
+    name = String()
+    address = String()
+    city_id = Integer()
```
Comment (frostming) [inline backend/bamboo/schemas/venue.py:16]: The out schema should contain all fields, including `updated_at` and `created_at`, may need to create a base schema for this.

Comment (180909) [reply backend/bamboo/schemas/venue.py:16]: Done

File: backend/bamboo/blueprints/venue.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    return venue
+
+
+@venue.delete("/<int:venue_id>")
+@venue.output({}, status_code=204)
+def delete_venue(venue_id):
+    venue = Venue.query.get_or_404(venue_id)
+    db.session.delete(venue)
+    db.session.commit()
+    return ""
```
Comment (frostming) [inline backend/bamboo/blueprints/venue.py:43]: Missing an endpoint to list all schedule_items associated to a specific venue, the route should be `/<int:venue_id>/schedules`

File: backend/bamboo/blueprints/venue.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+from bamboo.database.models import Venue
+from bamboo.schemas.venue import VenueIn, VenueOut
+
+venue = APIBlueprint("venue", __name__)
+
+
+@venue.get("/<int:venue_id>")
+@venue.output(VenueOut)
+def get_venue(venue_id):
+    return Venue.query.get_or_404(venue_id)
```
Comment (frostming) [inline backend/bamboo/blueprints/venue.py:13]: For this endpoint, we need to include the associated talk object, if any, in the output.

**Review by frostming — COMMENTED** [review]
Added some comments, one must consider how the API will be used by the CMS to decide what should be included in the input and output.

**Comment by frostming** [comment]
Please resolve the conflicts, we have merged some changes recently

File: backend/bamboo/blueprints/schedule_item.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+from bamboo.database.models import ScheduleItem
+from bamboo.schemas.schedule_item import ScheduleItemIn, ScheduleItemOut
+
+schedule_item = APIBlueprint("schedule_item", __name__)
+
+
+@schedule_item.get("/<int:schedule_item_id>")
+@schedule_item.output(ScheduleItemOut)
+def get_schedule_item(schedule_item_id):
+    return ScheduleItem.query.get_or_404(schedule_item_id)
```
Comment (frostming) [inline backend/bamboo/blueprints/schedule_item.py:13]:
```suggestion
    return db.get_or_404(ScheduleItem, schedule_item_id)
```

Please use the SQLAlchemy 2.0 style API: https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-orm-usage

**Review by frostming — CHANGES_REQUESTED** [review]

**Review by frostming — APPROVED** [review]

**Pull request merged**